dll/win32/browseui/shellbars/CISFBand.cpp

```diff
@@ -84,7 +88,7 @@ HRESULT CISFBand::CreateSimpleToolbar(HWND hWndParent)
     CComPtr<IEnumIDList> pEndl;    
     LPITEMIDLIST pidl;
     STRRET stret;     
-    hr = m_pISF->EnumObjects(0, SHCONTF_FOLDERS, &pEndl);
+    hr = m_pISF->EnumObjects(0, SHCONTF_FOLDERS|SHCONTF_NONFOLDERS, &pEndl);
```

Comment:
Isn't an ISFBand supposed to show folders, too?

Reply:
Yeah, this makes it show files as well.

Reply:
Oh. Duh. Yeah, I misread that.